Package.resolved

```diff
@@ -24,17 +24,17 @@
         "repositoryURL": "https://github.com/apple/swift-nio.git",
         "state": {
           "branch": null,
-          "revision": "98434c1f1d687ff5a24d2cabfbd19b5c7d2d7a2f",
-          "version": "1.13.0"
+          "revision": "87dbd0216c47ea2e7ddb1b545271b716e03b943e",
+          "version": "1.13.1"
         }
       },
       {
         "package": "swift-nio-http2",
         "repositoryURL": "https://github.com/apple/swift-nio-http2.git",
         "state": {
-          "branch": null,
-          "revision": "88bd6d369a729f88e63f53ae5265840de4bb1384",
-          "version": "0.2.1"
+          "branch": "nghttp2-support-branch",
```

Comment:
@lukasa we should probably make a new release here

Reply:
Makes sense; given that that branch will only receive minor patches in the future, I think it would make sense to depend directly on the branch regardless?

Reply:
@MrMage I _think_ that you cannot tag versions if you depend on anything by `.branch(...)`.

Reply:
@weissi sorry, what do you mean with that?

Reply:
@MrMage IIRC, if you have a package `foo` that depends on another package `bar` using `.branch("some-bar-branch")`, then `foo` can never have a tagged version.
IIRC the failure happens if I build an executable package `my-exec` that depends on `foo` with some version (say `from: "1.0.1"`) and the tagged `1.0.1` version depends on `bar` using `.branch("some-bar-branch")`, then `my-exec` won't compile.

Reply:
in other words I think whenever you plan to release a tagged version of something, you must not depend on anything else using `.branch("...")`.

Reply:
@weissi are you sure about that? I know that it works the other way around — if the package specifies a version and my binary target specifies a branch, the branch will simply take precedence (unless the two are completely incompatible). https://github.com/grpc/grpc-swift/pull/371 might be relevant to this as well.

Reply:
@MrMage the other way around works, yes. But what I described above does really not work, @aciid will know for sure and I think @tanner0101 does too?

Reply:
@MrMage I think the idea is that SemVer-wise it doesn't make sense to claim that you have a certain version if you're not restricting your dependency at all

Reply:
Thanks for the elaboration! In that case, we can probably drop this PR and have you guys tag 0.2.2 instead.

Reply:
@MrMage let's wait for @aciidb0mb3r or @tannner0101 to confirm, they know better

Reply:
I'm not sure whether or not you _can_ do that, but you definitely shouldn't.